### ISSUE TITLE
Updated: WCAG-EM Report Tool – Accessibility Evaluation Report Generator is "busy"

### ISSUE THREAD

**Issue by frahaliaman** [w3c/wai-wcag-em-report-tool#116]
**Describe the bug**
The link for the newly updated WCAG-EM Report Tool is not working. The only thing that comes up is the " Help improve this page" section of the page. When I listen to my NVDA, it says the page is "busy"
**To Reproduce**
1. Go to 'https://www.w3.org/WAI/news/2020-01-22/wcag-em-report-tool/"
2. Click on '[WCAG-EM Report Tool: Website Accessibility Evaluation Report Generator](http://www.w3.org/WAI/eval/report-tool/).'

**Expected behavior**
I should have been able to see the first step of the report: "Overview" along with the other steps.
**Screenshots**
<img width="1680" alt="WCAG-EM-Report-Tool 2022-10-16 13-09-50" src="https://user-images.githubusercontent.com/38879867/196029750-eb73a1c1-2889-4924-b1d4-2797db05433d.png">

**Desktop (please complete the following information):**
 - OS: [10.15.4]
 - Browser [chrome and firefox]
 - Version [Version 105.0.5195.125 (Official Build) (x86_64)]

**Comment by frahaliaman** [comment]
I believe the reason why it's not working is because I have another report open. Perhaps it won't work while I have that opened. Although, I tried it on another computer and has the same issue come up.
